content/en/observability_pipelines/advanced_configurations.md

```diff
@@ -26,7 +26,7 @@ further_reading:
 
 ## Overview
 
-This document goes over [bootstrapping the Observability Pipelines Worker](#bootstrap-options) and [referencing files in Kubernetes](#referencing-files-in-kubernetes).
+This document goes over bootstrapping the Observability Pipelines Worker.
```

Comment:
```suggestion
This document explains bootstrapping for the Observability Pipelines Worker.
```

Reply:
"goes over" is a kind of a colloquialism that might not translate well. Just a suggestion.